docs/projects/reverse-a-string/_posts/2019-10-16-dart.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Reverse a String in Dart
+layout: default
+last-modified: 2019-10-16
+featured-image:
+tags: [dart, reverse-a-string, string-reversal]
+authors:
+  - P. Ryon (slashdoom)
```

Comment:
Side note! Make sure you add yourself to the authors file. You'll need to come up with a username which you'll put here (slashdoom is fine).

https://github.com/TheRenegadeCoder/sample-programs/blob/master/docs/_data/authors.yml